### PR TITLE
Ufo2fdk should build heavy italic

### PR DESCRIPTION
**Pull request by moyogo**

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 from ufo2fdk import OTFCompiler
 from ufo2fdk.makeotfParts import MakeOTFPartsCompiler
 from robofab.world import OpenFont
-from itertools import islice
 import sys
 
 family = "CooperHewitt"
 weights = ["Thin", "Light", "Book", "Medium", "Semibold", "Bold", "Heavy"]
-for w in islice( weights, 0, len(weights)-1 ):
+for w in iter(weights[0:]):
```
Comment (adrientetar) [inline build.py]: Doesn’t the for loop in python call iter() by default?

Comment (moyogo) [inline build.py]: Yes, good point.

**Comment by adrientetar** [comment]
The next step is to contact Ellen Lupton on twitter to get PRs merged, she works at CH.

**Comment by moyogo** [comment]
Yeah maybe. I'm wondering if it would be easier to make a fork with a different font name.

**Pull request merged**